webapp/pages/settings/my-social-media.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <ds-card :header="$t('settings.social-media.name')">
+    <div>
+      <ds-input
+        v-model="value"
+        placeholder="Add social media url"
+        name="social-media"
+        :schema="{type: 'url'}"
+      />
+    </div>
+    <ds-space margin-top="base">
+      <div>
+        <ds-button
+          primary
+          @click="handleAddSocialMedia"
+        >
+          {{ $t('settings.social-media.submit') }}
+        </ds-button>
+      </div>
+    </ds-space>
+    <ds-space 
+      v-if="currentUser.socialMedia && currentUser.socialMedia.length"
+      margin-top="base"
+      margin="x-small"
+    >
+      <div
+        v-for="socialMediaIconUrl in currentUser.socialMedia"
+        :key="socialMediaIconUrl"
+      >
+        <a>
+          <img
+            :src="socialMediaIconUrl.match(/^(?:https?:\/\/)?(?:[^@\n])?(?:www\.)?([^:\/\n?]+)/g)[0] + '/favicon.ico'"
```

Comment:
I like the idea but do you know what happens if this doesn't work?

Reply:
yes, if there is some url that is shared, which does not have an endpoint `/favicon.ico`, then it will display a "broken" image